docs-dev/QUICKSTART.md

```diff
@@ -0,0 +1,32 @@
+# Launch a Ceramic Node
+
+This document is a quick start for launching a Ceramic node.  It is recommended that node operators read the full documentation at [developers.ceramic.network](https://developers.ceramic.network/)
+
+## PRE-REQUISITES
+
+* Node `v16`
+* npm `v8`
+
+## Install Ceramic
+
+`npm install -g @ceramicnetwork/cli`
+
+## Launch a Ceramic node
+
+This command will start a local JS Ceramic node connected to the Clay Testnet at https://localhost:7007
+
+`ceramic daemon`
+
+## Configuration
+
+The default configuration file is located at 
+
+`$HOME/.ceramic/daemon.config.json`
+
+For more information about the configuration options see [Server Configurations](https://composedb.js.org/docs/0.5.x/guides/composedb-server/server-configurations)
+
+For an example how to run in the cloud using Digital Ocean and Kubernetes see [Running in the Cloud](https://composedb.js.org/docs/0.4.x/guides/composedb-server/running-in-the-cloud)
```

Comment:
I seems like we are pointing to the 0.5x version of the ComposeDB here. Can we use the latest on here and point to the 0.5.x just like int the link right above this one?

Reply:
fixed!